pkgs/top-level/all-packages.nix

```diff
@@ -7424,7 +7424,7 @@ with pkgs;
   # 28 is pre-release
   beamMinimal28Packages = dontRecurseIntoAttrs beam_minimal.packages.erlang_28.beamPackages;
 
-  erlang_language_platform = callPackage ../by-name/er/erlang-language-platform/package.nix { };
+  erlang-language-platform = callPackage ../by-name/er/erlang-language-platform/package.nix { };
```

Comment:
```suggestion
```

This is failing the vet check because packages in by-name are automatically added. Hence why there were two packages before. You should completely remove from this file.

Reply:
Thanks for clarifying.